Core/Utilities/MQConnector.py

```diff
@@ -0,0 +1,103 @@
+"""
+Abstract class for management of MQ connections
+"""
+
+__RCSID__ = "$Id$"
+
+import time
+from DIRAC import gLogger, gConfig, S_OK, S_ERROR
+
+class MQConnector( object ):
+  """
+  Abstract class for management of message queue connections
+  Allows to both send and receive messages from a queue
+  """
+
+  def __init__( self ):
+    raise NotImplementedError( 'Cannot instantiate an abstract class' )
+
+  # Callback functions for message receiving mode
+
+  def on_error( self, headers, message ):
+    """
+    Callback function called when an error happens
+    """
+    gLogger.error( message )
+
+  # Rest of functions
+
+  def setQueueParameters( self, system, queueName ):
+    """
+    Reads the MessageQueue parameters from the CS and sets the appropriate variables in the class with their values
+    system is the DIRAC system where the queue works
+    queueName is the name of the queue
+    """
```

Comment:
@wkrzemien @miloszz 
On a second thought, after consultation, we agreed that this function should be partially re-written: in fact, the only parameter that should be get from the CS is the password. All the others should be passed in, e.g. by  ConsumerModule, that should discover them.